lib/holdings/item.rb

```diff
@@ -34,7 +33,8 @@ def browsable?
     end
 
     def on_order?
-      current_location.code == 'ON-ORDER'
+      folio_status == 'On order' ||
+        current_location.code == 'ON-ORDER'
```

Comment:
Is there a reason we still need to support the location code?

Reply:
At least until we do a full re-index with https://github.com/sul-dlss/searchworks_traject_indexer/pull/1275, on-order items without a FOLIO item don't have the status yet.

Reply:
Are we going to be able to remember that? Should we drop a comment here or create a new ticket that points to that?